examples/typescript-server.ts

```diff
@@ -0,0 +1,52 @@
+
+/**
+ * Most type annotations in this file are not strictly necessary but are
+ * included for this example.
+ * 
+ * To run this example exectute the following commands to install typescript,
+ * transpile the code, and start the server:
+ * 
+ * npm i -g typescript
+ * tsc examples/typescript-server.ts --target es6 --module commonjs
+ * node examples/typescript-server.js
+ */
+
+import * as fastify from '../fastify'
+import * as cors from 'cors'
+import { createReadStream } from 'fs'
+
+const server: fastify.FastifyInstance = fastify()
+
+const opts: fastify.RouteShorthandOptions = {
+  schema: {
+    response: {
+      200: {
+        type: 'object',
+        properties: {
+          hello: {
+            type: 'string'
+          }
+        }
+      }
+    }
+  }
+}
+
+function getHelloHandler (req: fastify.FastifyRequest, reply: fastify.FastifyResponse) {
```

Comment:
We should be consistent in naming - `reply: fastify.FastifyReply`, not response.

Reply:
I'm using express to much 😄 Fixed